### ISSUE TITLE
Rockbiter weapon has stopped working

### ISSUE THREAD

**Issue by Shadownflame** [Project-Epoch/bugtracker#8409]
All ranks at any level with any kind of weapon. Probably last update.

**Comment by kaytotes** [comment]
Next deploy

**Comment by Shadownflame** [comment]
Still broken.

**Comment by Shadownflame** [comment]
Correction, now it works if there is no other temporary enchantment present, but fails to apply if your weapon already has an effect.

**Comment by kaytotes** [comment]
Testable.

**Comment by Shadownflame** [comment]
After last fix, rockbiter can now override other enchant on the MH if the shaman is dual wielding. Doesn't work if you use a 2h, doesn't work if you have a 1h equipped with empty offhand slot, and if you have an 1h equipped with an offhand or shield, rockbiter will mercilessly buff the offhand/shield as well for some reason.
![WoWScrnShot_121624_182232](https://github.com/user-attachments/assets/99f58897-fa71-4778-95d8-ba61218f02d4)

**Comment by Hatson22** [comment]
Yep, still bugged like this. Not gameplay breaking, but quite annoying. I think Shaman players will start making /cancelaura macros for their imbues as long as these weird imbue rebuffing quirks exist.
